src/map/trait.h

```diff
@@ -139,7 +139,11 @@ enum TRAITTYPE
     TRAIT_PRIMEVAL_ZEAL       = 120,
     TRAIT_INSPIRATION         = 121,
     TRAIT_SLEIGHT_OF_SWORD    = 122,
-    TRAIT_DAKEN               = 123
+    TRAIT_DAKEN               = 123,
+    TRAIT_SUPERIOR            = 124, 

```

Comment:
Thats fine, if its a trait its a trait. If it makes sens internally to code is **as** as trait, gerat. Just don't expect being in a file to mean that. Theres a ton of non traits, same file.